frontend/src/pages/Candidate.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ProfileCard from '../components/ProfileCard/ProfileCard';
+import './style/Candidate.css'
+
+function Candidate() {
+  return (
+    <div className="container mt-4">
+      {/* Panel 1: AI Selected Candidates */}
+      <div className="row mb-4">
+        <div className="col">
+          <h3>AI Selected Candidates</h3>
+          <div
+            className="d-flex justify-content-start"
+            style={{
+              backgroundColor: '#d4edda',
```

Comment:
can we provide the styling via classes

Reply:
Completed this change in commit [3d7b685] . Moved the inline styles to the CSS class.